pkg/secrets-store/secrets-store.go

```diff
@@ -27,16 +27,20 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// SecretsStore implements the IdentityServer, ControllerServer and
+// NodeServer CSI interfaces.
 type SecretsStore struct {
 	driver *csicommon.CSIDriver
 	ns     *nodeServer
 	cs     *controllerServer
+	ids    *identityServer
 }
 
 var (
 	vendorVersion = "0.0.3"
```

Comment:
Just realized this should have been updated for each release.

Reply:
done! updated to reflect latest version. Going forward we can bump this along with other places before cutting new release.